packet-manager-default.properties

```diff
@@ -32,6 +32,12 @@ object.store.s3.url=http://minio.minio:9000
 object.store.s3.region=${s3.region}
 object.store.s3.readlimit=10000000
 
+
+##------------------------------------------------------Redis cache properties----------------------------------------------------
+redis.cache.hostname=10.20.20.90
+redis.cache.port=6379
+# Time to live for 2hrs
+spring.cache.redis.time-to-live=7200000
```

Comment:
Don't we need some property to enable cache to use redis like spring.cache.type=redis ?
Can we check on this from the country production config ?

Reply:
no we dont. Actually the implementation jar provided to packetmanager decides which one to use.
If we provide hazelcast cache jar then it will be used else redis is used as we pass that runtime.